assets/js/modules/analytics-4/datastore/__fixtures__/available-audiences.json

```
@@ -0,0 +1,38 @@
+[
+    {
+        "name": "properties/12345/audiences/1",
+        "description": "All users",
+        "displayName": "All Users",
```

@ankitrox, also the `displayName` here should be `"All visitors"` to reflect how it will be mapped as per the related change in this PR:

```suggestion
        "displayName": "All visitors",
```